### PR TITLE
[release-4.13] WINC-1020: Update version to 8.0.1

### PR DESCRIPTION
**Pull request by wgahnagl**
This commit was generated by hack/pre-release.sh

### CODE REVIEW

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -6,9 +6,9 @@ metadata:
     capabilities: Seamless Upgrades
     categories: OpenShift Optional
     certified: "false"
-    createdAt: REPLACE_DATE
+    createdAt: "2023-05-22T16:06:59Z"
```
Comment (saifshaikh48) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:9]: why is this being changed?

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -6,9 +6,9 @@ metadata:
...
     description: An operator that enables Windows container workloads on OCP
-    olm.skipRange: '>=7.0.0 <8.0.0'
+    olm.skipRange: '>=8.0.0 <8.0.1'
```
Comment (saifshaikh48) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: why bump the lower end of the skipRange here? what are the implications of this change?

**Review by saifshaikh48 — COMMENTED** [review]
Few comments. The PR should target the release branch rather than community btw

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (mtnbikenc) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:9]: In other words, this should not be changed.  This date is updated by `render_templates` in [dist-git](https://pkgs.devel.redhat.com/cgit/containers/windows-machine-config-operator-bundle/tree/render_templates?h=rhaos-4.13-rhel-9&id=b6b1019ef95ba89e9537535f5dc1cbfafba2f0a4#n60).

Comment (wgahnagl) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:9]: It seems that all of the changes being made by make bundle aren't needed, but is mentioned in the WMCO releasing doc as part of the post release tasks. Should we keep running make bundle and just ignore the changes it makes to the files, or remove it from the script entirely?

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (wgahnagl) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: you mentioned [here](https://github.com/openshift/windows-machine-config-operator/pull/1566#discussion_r1176816796) that that's the expected functionality of the script, and these changes are from that script.

Comment (saifshaikh48) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: imagine we were to run this script again with current functionality to prepare for version `8.0.2` -- the skipRange would be bumped to '>=9.0.0 <8.0.2' which doesn't look right
I suggest looking at [OLM upgrade process](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md), and how the skipRange looks in [other WMCO branches](https://github.com/openshift/windows-machine-config-operator/blob/release-4.11/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml)

Comment (wgahnagl) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: the way the bump works is that it takes the current version and puts it at the lower range, and then sets the updated version to the higher range. On an 8.0.2 bump, the skipRange would be >=8.0.1 <8.0.2
I can add a condition to only update that for the x version though. Should the lower range stay the same on minor versions?

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -367,7 +367,9 @@ spec:
           - use
         serviceAccountName: windows-machine-config-operator
       deployments:
-      - name: windows-machine-config-operator
+      - label:
+          name: windows-machine-config-operator
+        name: windows-machine-config-operator
```
Comment (alinaryan) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:372]: Are these supposed to be here twice?

**Review by alinaryan — COMMENTED** [review]
looking good other than my comment

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (mansikulkarni96) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml]: yes it should stay the same for minor versions, in short we do not need to touch skiprange for a minor version bump unless there is a need to do so.

**Comment by mtnbikenc** [comment]
/retitle [release-4.13] WINC-1020: Update version to 8.0.1

**Review by aravindhp — APPROVED** [review]
/lgtm
/hold cancel

**Comment by aravindhp** [comment]
/hold cancel

There is no ordering issue

**Pull request merged**